src/Control/Inciter/Options/Problem.h

```diff
@@ -26,6 +26,8 @@ enum class ProblemType : uint8_t { USER_DEFINED=0,
                                    SHEAR_DIFF,
                                    DIR_NEU,
                                    VORTICAL_FLOW,
+                                   NL_ENERGY_GROWTH,
```

Comment:
Its your project and your code. Do what you want. I am just providing the code review that was requested of me